.gitpod.yml

```diff
@@ -1,5 +1,6 @@
 tasks:
-  - init: npm install
+  - before: npm install
+    init: npm run build --workspaces
```

Comment:
`before` is meant to install something globally. In this case I'd write `init: npm install && npm run build --workspaces`

Reply:
oh ok sorry, i didn't knew that